packages/calcite-components/src/components/button/button.scss

```diff
@@ -4,122 +4,51 @@
  * @prop --calcite-button-box-shadow: Specifies the box shadow of the component.
- * @prop --calcite-button-box-shadow-hover: Specifies the box shadow of the component when in hover state.
- * @prop --calcite-button-box-shadow-focus: Specifies the box shadow of the component when in focus state.
- * @prop --calcite-button-box-shadow-active: Specifies the box shadow of the component when in active state.
  * @prop --calcite-button-border-color: Specifies border colors of the component.
- * @prop --calcite-button-border-color-hover: Specifies border colors of the component when in hover state.
- * @prop --calcite-button-border-color-focus: Specifies border colors of the component when in focus state.
- * @prop --calcite-button-border-color-active: Specifies border colors of the component when in active state.
  */
 
 :host {
   --calcite-button-background-color: var(--calcite-color-background);
   --calcite-button-text-color: var(--calcite-color-text-inverse);
   --calcite-button-box-shadow: var(--calcite-shadow-none);
   --calcite-button-border-color: var(--calcite-color-transparent);
-  --calcite-button-corner-radius: var(--calcite-corner-radius-sharp);
+  --calcite-button-corner-radius: var(--calcite-corner-radius);
   --calcite-button-icon-start-color: var(--calcite-color-text-inverse);
   --calcite-button-icon-end-color: var(--calcite-color-text-inverse);
-
-  /* hover */
-  --calcite-button-text-color-hover: var(--calcite-color-text-inverse-hover);
-  --calcite-button-background-color-hover: var(--calcite-color-background);
-  --calcite-button-border-color-hover: var(--calcite-color-transparent);
-  --calcite-button-box-shadow-hover: var(--calcite-shadow-none);
-  --calcite-button-icon-start-color-hover: var(--calcite-color-text-inverse-hover);
-  --calcite-button-icon-end-color-hover: var(--calcite-color-text-inverse-hover);
-
-  /* focus */
-  --calcite-button-text-color-focus: var(--calcite-color-text-inverse-hover);
-  --calcite-button-background-color-focus: var(--calcite-color-background);
-  --calcite-button-border-color-focus: var(--calcite-color-transparent);
-  --calcite-button-box-shadow-focus: var(--calcite-shadow-none);
-  --calcite-button-icon-start-color-focus: var(--calcite-color-text-inverse-focus);
-  --calcite-button-icon-end-color-focus: var(--calcite-color-text-inverse-focus);
-
-  /* active */
-  --calcite-button-text-color-active: var(--calcite-color-text-inverse-press);
-  --calcite-button-background-color-active: var(--calcite-color-background);
-  --calcite-button-border-color-active: var(--calcite-color-transparent);
-  --calcite-button-box-shadow-active: var(--calcite-shadow-none);
-  --calcite-button-icon-start-color-active: var(--calcite-color-text-inverse-active);
-  --calcite-button-icon-end-color-active: var(--calcite-color-text-inverse-active);
+  --calcite-button-box-shadow: inset 0 0 0 1px var(--calcite-button-box-shadow-color);
```

Comment:
Shouldn't we allow this to be a full box shadow rule, defaulting to our `none` - so a user can override this with, say box-shadow-sm, or box-shadow-md (or whatever they need). Do we need to change our usage of box-shadow within this component to use border? How would I set the border color and the box-shadow in this case?